src/components/AccountHeader/AccountHeader.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { AvatarFace, Button, Header } from 'decentraland-ui'
+import { Props } from './AccountHeader.types'
+import './AccountHeader.css'
+
+const AccountHeader = (_props: Props) => {
+  return (
+    <Header size="large" className="AccountHeader">
+      <AvatarFace size="large" />
+      <div className="profile-text">
+        <div className="profile-name"> Satanas </div>
+        <div className="profile-description"> At the movies </div>
+      </div>
+      <div className="actions">
+        <Button primary inverted> Change Alias </Button>
+        <Button primary inverted> Edit avatar </Button>
```

Comment:
Since these copies are final, we can already add translations for them, like:

```tsx
import { t } from 'decentraland-dapps/dist/modules/translation/utils'

// ...

<Button primary inverted>{t('account_header.actions.change_alias')}</Button>
```

And we should add those translations on each locale json file in `src/locales`

Reply:
done! thanks!